### PR TITLE
CB-11710 The error message that is displayed in the UI when a DL crea…

### PR DESCRIPTION
**Pull request by akanto**
…tion fails due to misconfigured IAM roles or Managed Identities is a bit misleading and it must be aligned.

See detailed description in the commit message.

### CODE REVIEW

**Review by biharitomi — APPROVED** [review]

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/error/mapper/ClouderaManagerStorageErrorMapper.java [hunk 1]
```diff
@@ -76,11 +76,11 @@ private String awsError(CloudStorage cloudStorage) {
                         "This problem usually occurs due to cloud storage permission misconfiguration. " +
                         "Services on the cluster are using Data Access Role (%s) and Ranger Audit Role (%s) to write to the Ranger Audit location (%s), " +
                         "therefore please verify that these roles have write access to this location. " +
-                        "During Data Lake cluster creation, CDP Control Plane attaches Data Access Instance Profile (%s) to the IDBroker VM. " +
-                        "IDBroker will then use it to assume the Data Access Role and Ranger Audit Role, therefore Data Access Instance Profile (%s) " +
+                        "During Data Lake cluster creation, CDP Control Plane attaches Assumer Instance Profile (%s) to the IDBroker VM. " +
```
Comment (doktoric) [inline cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/error/mapper/ClouderaManagerStorageErrorMapper.java:79]: VM or Virtual Machine which one is nicer ?

**Review by doktoric — APPROVED** [review]
+1 onyl one question added

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/error/mapper/ClouderaManagerStorageErrorMapper.java [hunk 2]
```diff
@@ -97,19 +97,19 @@ private String azureError(CloudStorage cloudStorage) {
...
                         "Ranger Audit location (%s), therefore please verify that these roles have write access to this location. " +
-                        "During Data Lake cluster creation, CDP Control Plane attaches Data Access Assumer Identity (%s) to the IDBroker VM. " +
-                        "IDBroker will then use it to attach the other managed identities to the IDBroker VM, therefore Data Access Identity (%s) " +
+                        "During Data Lake cluster creation, CDP Control Plane attaches Assumer Identity (%s) to the IDBroker VM. " +
+                        "IDBroker will then use it to attach the other managed identities to the IDBroker VM, therefore Assumer Identity (%s) " +
                         "permissions must, at a minimum, allow to attach the Data Access Identity and Ranger Access Identity. " +
                         "Refer to Cloudera documentation at %s for the required rights.",
-                auditLocation, dataAccessRole, rangerAuditRole, auditLocation, managedIdentity, managedIdentity,
+                auditLocation, dataAccessIdentity, rangerAuditIdentity, auditLocation, managedIdentity, managedIdentity,
```
Comment (lajosrodek) [inline cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/error/mapper/ClouderaManagerStorageErrorMapper.java:112]: I suggest renaming `managedIdentity` to `assumerIdentity` for consistency.

Comment (akanto) [reply cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/error/mapper/ClouderaManagerStorageErrorMapper.java]: Thanks. Makes sense.

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/error/mapper/ClouderaManagerStorageErrorMapper.java (hunk 1 above)
Comment (akanto) [reply cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/error/mapper/ClouderaManagerStorageErrorMapper.java]: Maybe VM is not obvious for everybody. I have updated to Virtual Machine.

**Comment by akanto** [comment]
Forget to mention that this is the UI counterpart https://github.com/hortonworks/hortonworks-cloud/pull/2667 (eventually landed in 2.40 as well)

**Pull request merged**